tests/test_examples.py

```diff
@@ -57,6 +57,7 @@ def powerset(iterable):
             "--no-make-video",
         ),
         ("examples/tutorials/semantic_id_tutorial.py", "--no-show-images"),
+        ("examples/tutorials/VHACD_tutorial.py", "--no-show-video", "--no-make-video"),
```

Comment:
Thanks for adding this. Will this error if vhacd is not enabled though? IF so we may need an additional skipif.

Reply:
Yeah, but CI should always hit this with --vhacd currently. 

Considering adding a binding like [this one for bullet in js](https://github.com/facebookresearch/habitat-sim/blob/f568b6564e59269e69f995ad54263b845dac1e22/src/esp/bindings_js/bindings_js.cpp#L127) as suggested by @eundersander.

Reply:
@aclegg3 It already exists: https://github.com/facebookresearch/habitat-sim/blob/f568b6564e59269e69f995ad54263b845dac1e22/habitat_sim/__init__.py#L62

Reply:
Thanks 👍